src/main/java/uk/gov/digital/ho/hocs/entrypoint/ProcessingController.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.digital.ho.hocs.entrypoint;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.digital.ho.hocs.domain.service.ProcessingService;
+import uk.gov.digital.ho.hocs.entrypoint.model.ProcessingRequest;
+
+@RestController
+public class ProcessingController {
+
+    private final ProcessingService processingService;
+
+    public ProcessingController(ProcessingService processingService) {
+        this.processingService = processingService;
+    }
+
+    @PostMapping("/process")
+    public ResponseEntity<Void> processMessages(@RequestBody ProcessingRequest processingRequest) {
+        processingService.processMessages(processingRequest.getMaxMessages(),
```

Comment:
Might be beneficial adding some log lines here to indicate the start and end? Although might be nice as a table?

Reply:
Logging would be useful just to see that it ran and with what parameters?

Reply:
Typically bad practice logging out request params, unsure if it'll allow them because its just dates and a long. At a minimum we should have start and end time logged for these.